### PR TITLE
OSOE-388: Release all projects

### LINKED ISSUE #216: Release all projects (OSOE-388)
Once https://github.com/Lombiq/UI-Testing-Toolbox/issues/101 is done, issue NuGet releases of all of the OSOCE projects.

[Jira issue](https://lombiq.atlassian.net/browse/OSOE-388)

### PR DESCRIPTION
**Pull request by Piedone**
OSOE-388
Fixes #216

### CODE REVIEW

**Review by dministro — CHANGES_REQUESTED** [review]
Lombiq.Tests was not released.
https://github.com/Lombiq/UI-Testing-Toolbox/blob/5773111eac4447090de29c454972ba0d162c1f42/Lombiq.Tests.UI/Lombiq.Tests.UI.csproj#L91

File: NuGetTest/test/Lombiq.OSOCE.NuGet.Tests/Lombiq.OSOCE.NuGet.Tests.csproj [hunk 1]
```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Lombiq.HelpfulLibraries" Version="4.6.1-alpha.walma-15.1" />
+    <PackageReference Include="Lombiq.HelpfulLibraries" Version="4.7.0" />
     <PackageReference Include="Lombiq.Tests" Version="2.0.1-alpha.osoe-46.1" />
```
Comment (dministro) [inline NuGetTest/test/Lombiq.OSOCE.NuGet.Tests/Lombiq.OSOCE.NuGet.Tests.csproj]: Lombiq.Tests was not released

**Review by dministro — CHANGES_REQUESTED** [review]

File: NuGetTest/test/Lombiq.OSOCE.NuGet.Tests/Lombiq.OSOCE.NuGet.Tests.csproj (hunk 1 above)
Comment (Piedone) [reply NuGetTest/test/Lombiq.OSOCE.NuGet.Tests/Lombiq.OSOCE.NuGet.Tests.csproj]: Eh.

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj [hunk 2]
```diff
@@ -12,27 +12,27 @@
...
+    <PackageReference Include="Lombiq.Hosting.Tenants.Management" Version="2.3.1" />
+    <PackageReference Include="Lombiq.JsonEditor" Version="2.2.0" />
+    <PackageReference Include="Lombiq.OrchardCoreApiClient" Version="1.1.0" />
+    <PackageReference Include="Lombiq.Privacy" Version="3.2.1" />
+    <PackageReference Include="Lombiq.Privacy.Samples" Version="3.2.1" />
     <PackageReference Include="Lombiq.SetupExtensions" Version="2.1.0" />
-    <PackageReference Include="Lombiq.Tests.UI.Shortcuts" Version="3.5.1-alpha.osoe-46.6" />
-    <PackageReference Include="Lombiq.UIKit" Version="2.1.1-alpha.osoe-46.7" />
+    <PackageReference Include="Lombiq.Tests.UI.Shortcuts" Version="4.0.1" />
+    <PackageReference Include="Lombiq.UIKit" Version="2.3.1" />
```
Comment (dministro) [inline NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:35]: Note: the latest release was 2.1.0.
https://www.nuget.org/packages/Lombiq.UIKit/2.1.0#versions-body-tab

**Review by dministro — APPROVED** [review]

File: NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj (hunk 2 above)
Comment (Piedone) [reply NuGetTest/src/Lombiq.OSOCE.NuGet.Web/Lombiq.OSOCE.NuGet.Web.csproj:35]: Oops. I perhaps got mixed up with the alpha releases (which BTW shouldn't be 2.2.x).

**Pull request merged**